README.md

```diff
@@ -93,6 +93,7 @@ and attaching them to your wofklow and releases.
 | `path`          | A path on the filesystem to scan. This is mutually exclusive to `image`.                                | \<current directory>        |
 | `image`         | A container image to scan. This is mutually exclusive to `path`.                                        |
 | `artifact-name` | The name to use for the generated SBOM artifact. See: [Naming the SBOM output](#naming-the-sbom-output) | `sbom-<job>-<step-id>.spdx` |
+| `format`        | The SBOM format to export. One of: `spdx`, `spdx-json`, `cyclonedx`, `table`, `text`, `json`            | `spdx-json`                 |
```

Comment:
Curious for your thoughts...

1. Do we want to expose Syft's output format options directly? (i.e. if Syft changes its list of choices, that change immediately flows through to this integration?)
2. If so, do we want to expose all options, or just a subset initially?

Reply:
I had initially thought to just expose standardized formats: `spdx`, `spdx-json`, and `cyclonedx`. But we can't perform a vulnerability scan on those, as was pointed out yesterday. Perhaps we just document those 3? Also, I'm not entirely sure about the naming. It seems fine unless we'd want to maybe do something like: `spdx/tag-value`, `spdx/json`. I really don't have a strong opinion here.

Reply:
> [...] we can't perform a vulnerability scan on those, as was pointed out yesterday

That's a good point. I'm not sure what the right answer is here. Today we can only use the `json` (proprietary) format. Our plan is to add support for the SPDX format(s) in the very near future. We don't plan to support vuln scanning for `text` and `table`, does that mean we rule these out as options?

> Also, I'm not entirely sure about the naming. It seems fine unless we'd want to maybe do something like: spdx/tag-value, spdx/json. I really don't have a strong opinion here.

Same here. I don't necessarily think names like `spdx-json` are bad at all. I think we should just understand the consequences and be able to explain to users however "tightly" we end up coupling this list to Syft's own list.